### ISSUE TITLE
Error on nvimcom install - Argument has length 0

### ISSUE THREAD

**Issue by mriedl93** [jalvesaq/Nvim-R#551]
I'm facing issues when trying to start the R server inside Neovim as the installation of nvimcom fails.
I'm getting the following error:
```
* installing to library ‘/home/markus/anaconda3/envs/default/lib/R/library’
* installing *source* package ‘nvimcom’ ...
staged installation is only possible with locking
** using non-staged installation
** libs
Fehler in if (nzchar(SHLIB_LIBADD)) SHLIB_LIBADD else character() : 
  Argument hat Länge 0
* removing ‘/home/markus/anaconda3/envs/default/lib/R/library/nvimcom’
```
The message is in german and translates roughly to "Error in if.... : Argument has length 0"

As can be seen in the error message I have R (3.6.1) managed in an anaconda (4.8.5) environment. I have neovim v0.5. When building (`R CMD build`) and installing (`R CMD INSTALL`), I get the exact same error upon installation.
Unfortunately, I couldn't find any similar issue here on Github or via Google. I also read the Package documentation on `:h Nvim-R` but found nothing that helped.

Thanks in advance for any help.

**Comment by jalvesaq** [comment]
Did you see https://github.com/jalvesaq/Nvim-R/issues/133 ? Is it helpful?

**Comment by staticnoisednb** [comment]
Yes, I saw that issue before. In the directory `~/.cache/Nvim-R/` there is only a file `path_to_nvimcom` as well as the README. Deleting the whole directory after switching to my anaconda environment didn't do the trick. Still get the exact same error and I think it's not the issue described in #133. However, I have a similar setup as described there. The environment with R is not my "base"/standard anaconda environment.

In the meantime I installed R on my machine the standard way as well, i.e. without conda management. It worked right away without any issues. I already suspected that anaconda is responsible for my problems. I don't know how satisfied I am with this workaround since I have to work with anaconda for a software project I'm working on, but for now I'm fine I guess.
